### ISSUE TITLE
Escape menu won't appear

### ISSUE THREAD

**Issue by Turbine2k5** [dhewm/dhewm3#331]
Pressing escape after a mission will sometimes bring up the console instead. I can't seem to see a pattern in how to replicate it. Using 1.5.1 RC2.

**Comment by DanielGibson** [comment]
maybe you're also pressing shift?
shift+escape opens the console

**Comment by Turbine2k5** [comment]
Nope, no keypresses other than escape. Although that makes me wonder if the game thinks I'm holding shift between levels. I'll see if I'm still running after it happens again.

**Comment by Turbine2k5** [comment]
Played around with it a bit. It seems to happen if I'm holding shift in the pause menu and load a game. Just pressing escape brings up the console, but depressing and releasing shift brings back the normal pause function.
